hedera-node/test-clients/src/main/java/com/hedera/services/bdd/spec/utilops/mod/QueryIdClearingStrategy.java

```diff
@@ -27,26 +27,29 @@
 import static java.util.Objects.requireNonNull;
 
 import edu.umd.cs.findbugs.annotations.NonNull;
+import java.util.EnumSet;
 import java.util.Map;
 
 public class QueryIdClearingStrategy extends IdClearingStrategy<QueryModification>
         implements QueryModificationStrategy {
     private static final Map<String, ExpectedAnswer> CLEARED_ID_ANSWERS = Map.ofEntries(
-            Map.entry("proto.ContractGetInfoQuery.contractID", ExpectedAnswer.onAnswerOnly(INVALID_CONTRACT_ID)),
-            Map.entry("proto.ContractCallLocalQuery.contractID", ExpectedAnswer.onAnswerOnly(INVALID_CONTRACT_ID)),
+            Map.entry("proto.ContractGetInfoQuery.contractID", ExpectedAnswer.onCostAnswer(INVALID_CONTRACT_ID)),
+            Map.entry("proto.ContractCallLocalQuery.contractID", ExpectedAnswer.onCostAnswer(INVALID_CONTRACT_ID)),
             Map.entry("proto.CryptoGetAccountBalanceQuery.accountID", ExpectedAnswer.onAnswerOnly(INVALID_ACCOUNT_ID)),
-            Map.entry("proto.CryptoGetAccountRecordsQuery.accountID", ExpectedAnswer.onAnswerOnly(INVALID_ACCOUNT_ID)),
-            Map.entry("proto.CryptoGetInfoQuery.accountID", ExpectedAnswer.onAnswerOnly(INVALID_ACCOUNT_ID)),
-            Map.entry("proto.ContractGetBytecodeQuery.contractID", ExpectedAnswer.onAnswerOnly(INVALID_CONTRACT_ID)),
-            Map.entry("proto.FileGetContentsQuery.fileID", ExpectedAnswer.onAnswerOnly(INVALID_FILE_ID)),
-            Map.entry("proto.FileGetInfoQuery.fileID", ExpectedAnswer.onAnswerOnly(INVALID_FILE_ID)),
+            Map.entry("proto.CryptoGetAccountRecordsQuery.accountID", ExpectedAnswer.onCostAnswer(INVALID_ACCOUNT_ID)),
+            Map.entry("proto.CryptoGetInfoQuery.accountID", ExpectedAnswer.onCostAnswer(INVALID_ACCOUNT_ID)),
+            Map.entry("proto.ContractGetBytecodeQuery.contractID", ExpectedAnswer.onCostAnswer(INVALID_CONTRACT_ID)),
+            Map.entry("proto.FileGetContentsQuery.fileID", ExpectedAnswer.onCostAnswer(INVALID_FILE_ID)),
+            Map.entry("proto.FileGetInfoQuery.fileID", ExpectedAnswer.onCostAnswer(INVALID_FILE_ID)),
             Map.entry(
                     "proto.TransactionID.accountID",
-                    ExpectedAnswer.onAnswerOnly(INVALID_ACCOUNT_ID, INVALID_TRANSACTION_ID)),
-            Map.entry("proto.ConsensusGetTopicInfoQuery.topicID", ExpectedAnswer.onAnswerOnly(INVALID_TOPIC_ID)),
-            Map.entry("proto.TokenGetInfoQuery.token", ExpectedAnswer.onAnswerOnly(INVALID_TOKEN_ID)),
-            Map.entry("proto.ScheduleGetInfoQuery.scheduleID", ExpectedAnswer.onAnswerOnly(INVALID_SCHEDULE_ID)),
-            Map.entry("proto.NftID.token_ID", ExpectedAnswer.onAnswerOnly(INVALID_TOKEN_ID)),
+                    new ExpectedAnswer(
```

Comment:
This is the only place in the code where we have an `ExpectedAnswer` that satisfies both "cost answer" and "answer only".  What does it mean?